### PR TITLE
New Package: create-release

### PR DESCRIPTION
**Pull request by srtfisher**
Introduces a new package that will be tied into https://github.com/alleyinteractive/create-wordpress-plugin/.

The package will help facilitate and release new versions along with https://github.com/alleyinteractive/create-wordpress-plugin/pull/258. In a future version, this will integrate with a better CHANGELOG management solution (such as https://github.com/changesets/changesets) to keep the CHANGELOG up to date on projects.

### CODE REVIEW

**Review by benpbolton — CHANGES_REQUESTED** [review]
I'm sorry. I'm going to need some changes. (not really, just clearing the funky codereview)

**Review by benpbolton — DISMISSED** [review]

File: packages/create-release/src/helpers.ts [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+ * Upgrade the version number in the composer.json file.
+ */
+function upgradeComposerVersion(basePath: string, version: string): void {
+  try {
+    const contents = JSON.parse(
+      fs.readFileSync(`${basePath}/composer.json`, 'utf8'),
+    );
+
+    // Insert the version after the description/name index.
+    const insertIndex = Object.keys(contents).indexOf('description') || Object.keys(contents).indexOf('name');
```
Comment (mogmarsh) [inline packages/create-release/src/helpers.ts:177]: what if `version` already exists? Am I missing something here?

**Review by mogmarsh — APPROVED** [review]
one question, otherwise 👍

File: packages/create-release/src/helpers.ts (hunk 1 above)
Comment (srtfisher) [reply packages/create-release/src/helpers.ts:177]: Let me test for that, good call

**Pull request merged**